pkgs/top-level/all-packages.nix

```diff
@@ -11915,7 +11915,18 @@ with pkgs;
 
   linuxPackages_grsec_nixos =
     recurseIntoAttrs (linuxPackagesFor linux_grsec_nixos);
-
+    
+  # Grsecurity kernel for proprietary nvidia driver
+  linux_grsec_nvidia = linux_grsec_nixos.override (old: {
```

Comment:
I believe this mixes `override` and `overrideAttrs`; the former affects the parameters passed to the original expression; the latter affects the inner `mkDerivation` call. With `override` you typically pass a plain record parameter.